style.css

```diff
@@ -46,14 +52,93 @@ body {
 
 /* ===== bootswatch ===== */
 .theme-darkly .form-control, .theme-darkly .form-control:focus {
-    background-color: #303030;
+    background-color: #303030 !important;
     color: #fff;
 }
 
+/*Changes the color of the grey differences to be a bit lighter in dark theme*/
+.theme-darkly .text-secondary{
+	color: #bdbdbd !important; 
+}
+
+/*Fixes the selected header and sort arrows in the light theme*/
+.theme-flatly .ng-table th.sortable.sort-asc,.theme-flatly .ng-table th.sortable.sort-desc{
+	background-color: #e7e8e9;
+	color: #000;
+}
+
+.theme-flatly .ng-table th.sortable.sort-desc .sort-indicator::after {
+    border-top-color: #000;
+}
+
+.theme-flatly .ng-table th.sortable .sort-indicator::after, .theme-flatly .ng-table th.sortable .sort-indicator::before {
+    border-color: #000 transparent;
+}
+
+
 /* ===== custom table width ===== */
 @media (min-width: 992px) {
-    .min-col-width { width: 1%; }
+    .min-col-width { width: 10%; }
     .type-col-width { width: 80px; }
     .weapon-col-width { width: 120px; }
-    .number-col-width { width: 80px; }
+    .number-col-width { width: 90px; }
+}
+
+/*Makes the header sticky, slight visual issue on firefox*/
+.header{
+	position: -webkit-sticky;
+	position: sticky;
+	top: 0;	
+}
+.filter {
+	position: -webkit-sticky;
+	position: sticky;
+	top: 33px;
+
+}
+
+/* The header background color for the light theme*/
+.theme-flatly .header,.theme-flatly .filter{
+	background-color: #ffff;
+}
+
+/* The header background color for the dark theme*/
+.theme-darkly .header,.theme-darkly .filter{
+	background-color: #222;
+}
+
+/*Changes the border collapse to fix the borders on the fixed header, adds some borders for looks*/
+table{
+	border-collapse: separate; /* Don't collapse */
+	border-spacing: 0;
+}
+
+/*Light theme borders*/
+.theme-flatly th,.theme-flatly td{
+	border-left: 1px solid var(--light-border-color);
+}
+.theme-flatly table{
+	border-right: 1px solid var(--light-border-color);
+	border-bottom: 1px solid var(--light-border-color);
+}
+
+/*Dark theme borders*/
+.theme-darkly th,.theme-darkly td{
+	border-left: 1px solid var(--dark-border-color);
+}
+.theme-darkly table{
+	border-right: 1px solid var(--dark-border-color);
+	border-bottom: 1px solid var(--dark-border-color);
+}
+
+
+/* Centers some stuff*/
+.table th, .table td {
+    vertical-align: middle;
+}
+.min-col-width, .text-lg-right{
+	text-align: center !important;
```

Comment:
Aha, my original intention is that, numeric stuff are better right aligned, for calculation purposes. For example:

```
Total : 2,450.67
Profit:   128.90
```

But, you are correct in that we don't do calculations here. However, this is not the right way to change it. We can just change the HTML class, from `text-lg-right` to `text-lg-center`. They are utility CSS classes provided by the UI framework.

https://getbootstrap.com/docs/4.3/utilities/text/#text-alignment

Reply:
> Aha, my original intention is that, numeric stuff are better right aligned, for calculation purposes.

As I said it's a subjective thing, with the borders between the columns it looks fine with the right alignment.
![image](https://user-images.githubusercontent.com/49487896/56897891-32f7c400-6a7f-11e9-9764-f2062d01d71b.png)
Without the borders I'd say it looks as if the columns are miss aligned. For me the whole table looks better when the columns with the numbers are similar size and centred, it makes navigating the table easier. 
There isn't much point comparing numbers in the table besides when comparing the rerolled and unrolled prices but thats only 2 rows so them not being right aligned isn't problematic.

Reply:
Indeed, lets keep it center aligned 👍